docs-site/docs/graph.md

```diff
@@ -5,4 +5,412 @@ title: Computational Graph
 
 <!-- Licensed to the Apache Software Foundation (ASF) under one or more contributor license agreements.  See the NOTICE file distributed with this work for additional information regarding copyright ownership.  The ASF licenses this file to you under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the License for the specific language governing permissions and limitations under the License. -->
 
-SINGA supports buffering operation and computational graph.
+SINGA supports buffering operations and computational graph. Using computational graph, SINGA can schedule the execution of operations as well as the memory allocation and release. It makes training more efficient while using less memory.
+
+## Features
```

Comment:
* it seem the prettier tool didn't format this markdown file..
* I suggest to rearrange the content of this file
  1. explain what the computational graph is.  include an simple example figure?
  2. describe the features
  3. show the usage (example)
  4. present the experiment
  5. introduce the implementation (how the python code is executed at the CPP side, how the operations are buffered, and when and how they are executed) by referring to the example
  6. how to add a new operation.